tools/cut_a_release.md

```diff
@@ -0,0 +1,47 @@
+# Cutting a Deno release
+
+**During this process `main` branch should be frozen and no commits should land
+until the release is cut.**
+
+1. Create a PR that bumps versions of all crates in `op_crates` and `runtime`
+   directories.
+
+If this is a Deno patch release then the crates should have patch version bumped
+as well; analogously for a minor Deno release the crates should have minor
```

Comment:
This is not what @lucacasonato was saying.  Can we clarify this.

Reply:
Clarified it.